### DISCUSSION TITLE
Updating a field in one table in response to a change in a field in another

### DISCUSSION THREAD

**Discussion by taholmes160** [Budibase/budibase#9063] [General]
HI folks:  In this app for moving cars around -- I'd like to put a field on each car that contains its CURRENT location -- (that part is easy)  however whenever the car participates in a move, i'd like the location field to update based on where the move ended -- im not quite sure how to make that happen.  Im not afraid to do research and dig, im just not sure what I should be looking for -- can someone point me in the right direction?  -- I can send screen shots, or exported app or whatever (Not sure what you might need)

Thanks a bunch
TIM

**Comment by melohagan** [comment]
Hey @taholmes160 

Maybe [Automations](https://docs.budibase.com/docs/automation-steps) would help in this case?

- Trigger: **Row Update** (one table)
- Action: **Update Row** (another table)
